Dockerfile

```diff
@@ -50,71 +43,40 @@ RUN git rev-parse --short HEAD > ./VERSION
 RUN RAILS_ENV=production SECRET_KEY_BASE=required-to-run-but-not-used \
     bundle exec rails assets:precompile
 
-# Cleanup to save space in the production image
-RUN rm -rf node_modules log/* tmp/* /tmp && \
-    rm -rf /usr/local/bundle/cache && \
-    rm -rf .env && \
-    rm -rf .git && \
-    find /usr/local/bundle/gems -name "*.c" -delete && \
-    find /usr/local/bundle/gems -name "*.h" -delete && \
-    find /usr/local/bundle/gems -name "*.o" -delete && \
-    find /usr/local/bundle/gems -name "*.html" -delete
-
 RUN chown -R appuser:appgroup /app
 
 USER 1000
 
-# Build PDFTK
-FROM ghcr.io/graalvm/graalvm-ce:22.2.0 as pdftkbuilder
-RUN gu install native-image
-WORKDIR /build
-RUN curl https://gitlab.com/api/v4/projects/5024297/packages/generic/pdftk-java/v3.3.3/pdftk-all.jar --output pdftk-all.jar \
-	&& curl https://gitlab.com/pdftk-java/pdftk/-/raw/v3.3.3/META-INF/native-image/reflect-config.json --output reflect-config.json \
-	&& curl https://gitlab.com/pdftk-java/pdftk/-/raw/v3.3.3/META-INF/native-image/resource-config.json --output resource-config.json \
-	&& native-image --static -jar pdftk-all.jar \
-        -H:Name=pdftk \
-        -H:ResourceConfigurationFiles='resource-config.json' \
-        -H:ReflectionConfigurationFiles='reflect-config.json' \
-        -H:GenerateDebugInfo=0
-
-
 # Build runtime image
-FROM ruby:3.2.2-alpine as production
+FROM ruby:3.2.2-slim-bookworm as production
 
 # The application runs from /app
 WORKDIR /app
 
 # Add the timezone (prod image) as it's not configured by default in Alpine
-RUN apk add --update --no-cache tzdata && \
-    cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
-    echo "Europe/London" > /etc/timezone
-
-# libpq: required to run postgres
-RUN apk add --no-cache libpq postgresql-client
-
-# Install Chromium and Puppeteer for PDF generation
-# Installs latest Chromium package available on Alpine (Chromium 108)
-RUN apk add --no-cache \
-        chromium \
-        nss \
-        freetype \
-        harfbuzz \
-        ca-certificates \
-        ttf-freefont \
-        nodejs \
-        yarn
-
-# Tell Puppeteer to skip installing Chrome. We'll be using the installed package.
-ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
-ENV PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser
-
-# Install latest version of Puppeteer that works with Chromium 108
-RUN yarn add puppeteer@19.2.0
-
-# Copy files generated in the builder images
+#RUN apk add --update --no-cache tzdata && \
+#    cp /usr/share/zoneinfo/Europe/London /etc/localtime && \
+#    echo "Europe/London" > /etc/timezone
+
+RUN apt update
+# possibly don't need to specify all these sub-dependencies any more...
+# probably need ca-certificates so that chromium can talk to something
+RUN apt install -y postgresql-client nodejs fonts-freefont-ttf libharfbuzz-bin nss-tlsd pdftk ca-certificates
```

Comment:
Could we combine this to 

`RUN apt update & apt install -y postgresql-client nodejs fonts-freefont-ttf libharfbuzz-bin nss-tlsd pdftk ca-certificates`

We would combine lines 61 & 64